.travis.yml

```diff
@@ -31,14 +30,10 @@ matrix:
         - DEPS=lowest
     - php: 5.6
       env:
-        - EXECUTE_TEST_COVERALLS=true
+        - TEST_COVERAGE=true
         - DEPS=locked
         - DEPLOY_DOCS="$(if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then echo -n 'true' ; else echo -n 'false' ; fi)"
         - PATH="$HOME/.local/bin:$PATH"
-    - php: 5.6
-      env:
-        - PHPUNIT_VERSION=~5.0
-        - DEPS=latest
```

Comment:
I think we should test also with latest dependencies on PHP 5.6, but to do that we have to remove `--ignore-platform-reqs` in `COMPOSER_ARGS`. What do you think?